vpype_cli/operations.py

```diff
@@ -662,12 +662,21 @@ def layout(
     is_flag=True,
     help="Rotate clockwise instead of the default counter-clockwise",
 )
+@click.option(
+    "--orientation",
+    "-o",
+    type=click.Choice(["any", "portrait", "landscape"]),
+    default="any",
```

Comment:
- I think we can do without `any`. If we use that option, it means we want a specific orientation.
- Better use `ChoiceType` wrapper, so it supports substitution (`--orientation %some_var%`).

```suggestion
    type=ChoiceType(["portrait", "landscape"]),
```